wallet-extension/tutorials/get-started.md

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Shouldn't this be in docs?

Reply:
Ofc it should, but let's first review PR and make sure it's final. Once merged Shukhart will add it and we will remove from here